projects/pswg/src/main/java/com/parzivail/pswg/mixin/ItemRendererMixin.java

```diff
@@ -43,6 +50,24 @@ public void renderItem(LivingEntity entity, ItemStack stack, ModelTransformation
 		}
 	}
 
+	@ModifyVariable(method = "renderItem", at = @At(value = "HEAD"), argsOnly = true)
```

Comment:
totally remove all of this (including the other mixins you made) and use our item rendering framework. For reference, see [this line where lightsabers and blasters are registered](https://github.com/Parzivail-Modding-Team/GalaxiesParzisStarWarsMod/blob/master/projects/pswg/src/main/java/com/parzivail/pswg/Client.java#L312) and work from there. You'll have to export the BB model as a P3Di, the compile that with Aurek. ping me in Discord when you're ready to do this